debugtools/DDR_Artifacts/specs/29/linux_x86-64_cmprssptrs_panama

```diff
@@ -6055,7 +6055,7 @@ S|J9JavaAccessFlags|J9JavaAccessFlagsPointer|
 C|J9AccAbstract
 C|J9AccAnnotation
 C|J9AccBridge
-C|J9AccClassAnnnotionRefersDoubleSlotEntry
+C|J9AccClassAnnotationRefersDoubleSlotEntry
```

Comment:
@keithc-ca please advise should we be making the changes in DDR_Artifacts? I think these will be going away soon?

Reply:
None of the files in`debugtools/DDR_Artifacts` need be touched. The content of those files are not relevant to openj9 builds.

Reply:
ok. I'll add that set to ignore. thanks.